packages/components/helptext/test/HelpText.test.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
```

Comment:
`Tests are kept next to their components and use the .test.js file extension.` from the [README](https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components#testing). Please move it to the same folder as the component :)

Reply:
All new components have /test folder and thats a result from running `yarn generate` - guess that's another thing to discuss and align so we could either change what that generates or the README for v4.

Reply:
You're right. I was already about to file a PR that would reorganize all those test files but I'm gonna wait for a decision in the f36 channel.